specs/protocol/configurability.md

```diff
@@ -379,7 +379,7 @@ Address:
 
 **Description:** Account authorized to change values in the SystemConfig contract. All configuration is stored on L1 and
 picked up by L2 as part of the [derivation](./derivation.md) of the L2 chain.<br/>
-**Administrator:** <br/>
+**Administrator:** [L1 Proxy Admin](#admin-roles)<br/>
```

Comment:
noticed this was missing. Other changes in this file are either from linting or my IDE's formatting, but are purely cosmetic.